packages/models/organizations.coffee

```diff
@@ -4,6 +4,19 @@ Organization = Astro.Class
   collection: Organizations
   transform: true
   fields:
-    name: 'string'
-    description: 'string'
-    createdById: 'string'
+    name:
+      type: 'string'
+      validators: [
+        Validators.unique()
+        Validators.string()
+      ]
+    description:
+      type: 'string'
+      validators: [
+        Validators.string()
+      ]
+    createdById:
+      type: 'string'
+      validators: [
+        Validators.string()
```

Comment:
Is it possible to have a validator that checks whether the id matches the current user, or at least some user?